docs/source/changelog.rst

```diff
@@ -28,6 +28,7 @@ New features
 - Add support for Python 3.8 in :pr:`840` by :user:`minrk`.
 - Add Hydroshare as content provider in :pr:`800` by :user:`sblack-usu`.
 - Update to Jupyter Notebook 6 and Lab 1.2 in :pr:`839` by :user:`minrk`.
+- Update Nix from 2.3 to 2.3.9 in :pr:`991` by :user:`FRidh`.
```

Comment:
wrong release

Reply:
Like in your other PR: I don't quite understand why you added this comment to the change log for an already released version.